tools/mmp/Makefile

```diff
@@ -180,11 +180,11 @@ $(MMP_DIRECTORIES):
 GENERATE_PART_REGISTRAR = $(Q_GEN) $(SYSTEM_MONO) --debug mmp.exe --xamarin-framework-directory=$(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR) -q --runregistrar:$(abspath $@) --sdkroot $(XCODE_DEVELOPER_ROOT) --sdk $(OSX_SDK_VERSION)   $< --registrar:static
 
 Xamarin.Mac.registrar.mobile.i386.m:   $(TOP)/src/build/mac/mobile-32/Xamarin.Mac.dll $(LOCAL_MMP)
-	$(GENERATE_PART_REGISTRAR) --target-framework Xamarin.Mac,v1.0 --arch=i386
+	$(GENERATE_PART_REGISTRAR) --profile Xamarin.Mac,Version=v2.0,Profile=Mobile --arch=i386
```

Comment:
`--profile` is obsolete. `--target-framework` should be used instead (according to `mmp --help`).